internal/integration/provision/provision.go

```diff
@@ -71,3 +103,533 @@ func trimVersion(version string) string {
 	// remove anything extra after semantic version core, `v0.3.2-1-abcd` -> `v0.3.2`
 	return regexp.MustCompile(`(-\d+-g[0-9a-f]+)$`).ReplaceAllString(version, "")
 }
+
+var defaultNameservers = []netip.Addr{netip.MustParseAddr("8.8.8.8"), netip.MustParseAddr("1.1.1.1")}
+
+// BaseSuite provides base features for provision tests.
+type BaseSuite struct {
```

Comment:
no real changes, pulled out common part of `UpgradeSuite` into `BaseSuite` to re-use with Kubernetes version test